src/proxy-ui-api/frontend/nightwatch.json

```diff
@@ -31,8 +31,10 @@
         "test_cert" : "cert.cer",
         "export_cert" : "certs.tar.gz",
         "e2etest_testdata" : "/../testdata",
+        "testfiles_path" : "/var/lib/tomcat7/webapps/ROOT/test_data",
```

Comment:
Using this static path makes tests rely on this certain path. I think a better solution for testing wsdl-refresh would be to provide a http interface for switching the served wsdl description. Current solution may work but it feels overly constrictive if we think about setting up ephemeral testing environments. 

However I think we have some pressure to get these tests running so we could do the refactoring in later versions.